### ISSUE TITLE
How to get data from Tarantool using SQL query

### ISSUE THREAD

**Issue by mnvx** [tarantool-php/client#30]
I try to use next way

```php
$conn = new StreamConnection('tcp://' . env('TARANTOOL_HOST') . ':' . env('TARANTOOL_PORT'));

$packer = new PurePacker();
$client = new Client($conn, $packer);

$client->evaluate('box.cfg()');
$client->evaluate('box.sql.execute([[CREATE TABLE table1 (column1 INTEGER PRIMARY KEY, column2 VARCHAR(100));]])');
$client->evaluate('box.sql.execute([[INSERT INTO table1 VALUES (1, \'A\');]])');
$result = $client->evaluate('box.sql.execute([[SELECT * FROM table1;]])');
var_dump($result->getData());
```
As I see in Tarantool console, data was created:

```
tarantool> select * from table1;
---
- - [1, 'A']
...
```

But result in PHP is empty:
```
/var/www/cup-backend/app/Http/Controllers/Test/TarantoolController.php:26:
array (size=0)
  empty
```

What is wrong?

**Comment by mnvx** [comment]
And it works with next way...

```php
$client->evaluate('function testfunc()
    return box.sql.execute([[SELECT * FROM table1;]])
end');
$result = $client->call('testfunc');
var_dump($result->getData());
```

Result is:
```
array (size=1)
  0 => 
    array (size=1)
      0 => 
        array (size=2)
          0 => int 1
          1 => string 'A' (length=1)
```

**Comment by rybakit** [comment]
I haven't tested it myself, but it looks like the lua's `return` statement is missing in your first example. Try this line instead:

```php
$result = $client->evaluate('return box.sql.execute([[SELECT * FROM table1;]])'); // <- note return here
```

**Comment by mnvx** [comment]
Thanks! It works!

**Comment by artem-webdev** [comment]
Thank you very much for the library! 
I would not like to correct such mistakes - 

`PHP Fatal error:  Uncaught Error: Class 'MessagePack\Packer' not found in /home/work/php/test/vendor/tarantool/client/src/Packer/PurePacker.php:19
Stack trace:
#0 /home/work/php/test/tarantool.php(12): Tarantool\Client\Packer\PurePacker->__construct()
#1 {main}
  thrown in /home/work/php/test/vendor/tarantool/client/src/Packer/PurePacker.php on line 19

Fatal error: Uncaught Error: Class 'MessagePack\Packer' not found in /home/work/php/test/vendor/tarantool/client/src/Packer/PurePacker.php:19
Stack trace:
#0 /home/work/php/test/tarantool.php(12): Tarantool\Client\Packer\PurePacker->__construct()
#1 {main}
  thrown in /home/work/php/test/vendor/tarantool/client/src/Packer/PurePacker.php on line 19`

**Comment by rybakit** [comment]
Hi Artem,

Looks like the [rybakit/msgpack](https://packagist.org/packages/rybakit/msgpack) package is missing in your setup, check a note under the "Usage" section for details: https://github.com/tarantool-php/client#usage. 

TLDR, try to run `composer require rybakit/msgpack` to solve the issue.

**Comment by artem-webdev** [comment]
Yes, thank you very much, I did it.
I would like this dependency to be enabled by default!

**Comment by rybakit** [comment]
> I would like this dependency to be enabled by default!

Well, rybakit/msgpack is not necessary if you use the [msgpack-php](https://pecl.php.net/package/msgpack) pecl extension. That is why it's not enabled by default.

**Comment by artem-webdev** [comment]
how to get an associative array?

**Comment by artem-webdev** [comment]
var_dump($result->getData());  returns values without columns

**Comment by rybakit** [comment]
@artem-webdev Please ask that kind of questions on stackoverflow.com or at least create a dedicated issue, it will make easier to other concerned developers to find answers to these questions in the future.
